src/stream/src/executor/actor.rs

```diff
@@ -74,6 +78,7 @@ impl ActorContext {
         total_mem_val: Arc<TrAdder<i64>>,
         streaming_metrics: Arc<StreamingMetrics>,
         unique_user_errors: usize,
+        dispatch_num: usize,
```

Comment:
The name for this method seems to be outdated. Also, is `ActorContext::create` only supposed to be used by tests?

Reply:
Yes. I think `ActorContext::create` is only for test.

Reply:
Maybe we can rename the `create` to `for_test` and rename the `create_with_metrics` to `create` in the future.